test-plans/cmd/ping/main.go

```diff
@@ -200,3 +209,28 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func generateTLSConfig() *tls.Config {
+	priv, err := rsa.GenerateKey(rand.Reader, 2048)
```

Comment:
Any reason not use ed25519 here?

Reply:
I copied this from our existing generateTLSConfig func in tests